src/components/BulkMetadataForm/BulkMetadataFormStyles.ts

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { themeSpacing } from '@amsterdam/asc-ui';
+
+const BulkMetadataFormStyles = styled.div`
+	input[type='text'] {
+		margin: ${themeSpacing(3, 0, 6)};
+	}
+
+	.MuiGrid-item {
+		text-align: center;
+	}
+
+	.MuiGrid-spacing-xs-2 div:nth-of-type(2) {
+		div:first-of-type {
+			width: 100%;
+		}
+	}
+
+	.MuiCardContent-root .MuiGrid-container .MuiGrid-item .MuiGrid-container > div:first-of-type {
```

Comment:
You can reduce the amount of selector by combining this with the next block.
```
.MuiCardContent-root .MuiGrid-container .MuiGrid-item .MuiGrid-container > div {
    &:first-of-type {}
    &:nth-of-type(2) > label > span {}
}
```

Reply:
I see that another `.MuiGrid-container .MuiGrid-item .MuiGrid-container .MuiGrid-item` is used below. Better to combine it with that selector at the top level then I think.

Reply:
Combined selectors